boot/zephyr/CMakeLists.txt

```diff
@@ -347,6 +347,8 @@ if(NOT CONFIG_BOOT_SIGNATURE_TYPE_NONE AND NOT CONFIG_BOOT_SIGNATURE_KEY_FILE ST
     set(signing_key_file ${key_file})
   elseif(EXISTS ${APPLICATION_CONFIG_DIR}/${key_file})
     set(signing_key_file ${APPLICATION_CONFIG_DIR}/${key_file})
+  elseif((DEFINED WEST_TOPDIR) AND (EXISTS ${WEST_TOPDIR}/${key_file}))
```

Comment:
Not acceptable, what you mention is the signing for the application which is relative to the application. Instead of doing this, use sysbuild and use an escaped path e.g. `\${APP_DIR}/my.pem`